modules/Synapse/workspaces/integrationRuntimes/main.bicep

```diff
@@ -0,0 +1,63 @@
+@description('Conditional. The name of the parent Synapse Workspace. Required if the template is used in a standalone deployment.')
+param workspaceName string
+
+@description('Required. The name of the Integration Runtime.')
+param name string
+
+@allowed([
+  'Managed'
+  'SelfHosted'
+])
+@description('Required. The type of Integration Runtime.')
+param type string
+
+@description('Optional. The name of the Managed Virtual Network if using type "Managed" .')
```

Comment:
```suggestion
@description('Conditional. The name of the Managed Virtual Network. Required if using type "Managed" .')
```

Reply:
removed this parameter since it's not required anymore. the managed virtual network name is ALWAYS `default` if created